src/Sylius/Bundle/AdminBundle/templates/product_variant/form/content/update/header/actions/price_history.html.twig

```diff
@@ -0,0 +1,35 @@
+{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon %}
+
+{% set product_variant = hookable_metadata.context.resource %}
+
+{% if product_variant.channelPricings|length < 1 %}
```

Comment:
I guess it should be 🤔:
```suggestion
{% if product_variant.channelPricings|length == 1 %}
```
or
```suggestion
{% if product_variant.channelPricings|length < 2 %}
```

Reply:
I revised the logic slightly to display the list even if there is only one channel. For fewer than one channel, I disabled the button.